recipes/mplfinance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mplfinance" %}
+{% set version = "0.12.7a0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e7a6f80a0386419b184434dfc74a943f55f86baa906c25fe2a8d338d5edce62
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.6
+  host:
+    - matplotlib
+    - pandas
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
+    - pandas
+    - python >=3.6
+
+test:
+  imports:
+    - mplfinance
+    - mplfinance._styledata
+
+about:
+  home: "http://github.com/matplotlib/mplfinance"
+  license: BSD
+  license_family: BSD
+  license_file: "LICENSE"
```

Comment:
The license file is missing from the source distribution. I opened a PR upstream to address this. Either you'll have to wait for that to be merged and then a new release pushed to pypi, or you can manually package a copy now:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

and then fix it in the feedstock once the license file is in the source distribution.

Reply:
Thanks.  I've merge the PR.   Will update pypi at a later date.  In the meantime have copied the license into the recipe/mplfinance folder.